### DISCUSSION TITLE
Doc.md usability / ux

### DISCUSSION THREAD

**Discussion by olup** [vlang/v#11756] [Ideas]
Being a fresh learner of v, the doc.md in this repo is a very important first source of information, as it brings guides and example I could not figure out just with the lib documentation. I would love to be able to navigate this doc.md in a more user friendly way. Maybe search it, maybe open the examples in the playground (at least the simple ones).

Maybe this initial documentation could be static-file generated and live on the main website, in a docusaurus way ?

**Comment by ctkjose** [comment]
We recently update [V-Docs](https://docs.vlang.io)  the UX more up-to-date. Im also working on an open book at v-lang.dev that will be live feed from a git repository and will allow for translations of the book with a much easier process to update the books.  Hope to upload the WIP soon.
